app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -0,0 +1,102 @@
+        enableToolbarBackButton()
+
+        val copyright = binding.creditsSub
+        copyright.text = content.copyright
+        val versionHistory = binding.versionHistorySub
+        versionHistory.text = content.versionHistory
+
+        binding.versionHistory.tag = "https://catima.app/changelog/"
+        binding.translate.tag = "https://hosted.weblate.org/engage/catima/"
+        binding.license.tag = "https://github.com/CatimaLoyalty/Android/blob/master/LICENSE"
+        binding.repo.tag = "https://github.com/CatimaLoyalty/Android/"
+        binding.privacy.tag = "https://catima.app/privacy-policy/"
+        binding.reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
+        binding.rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"
+
+        bindClickListeners()
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        val id = item.itemId
+        if (id == android.R.id.home) {
+            finish()
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        content?.destroy()
+        content = null
+        clearClickListeners()
+        binding = null
+    }
+
+    private fun bindClickListeners() {
+        val binding = binding!!
+
+        val openExternalBrowser = View.OnClickListener { view: View ->
+            val tag = view.tag
+            if (tag is String && tag.startsWith("https://")) {
+                OpenWebLinkHandler().openBrowser(this, tag)
+            }
+        }
+
+        binding.versionHistory.setOnClickListener(openExternalBrowser)
+        binding.translate.setOnClickListener(openExternalBrowser)
+        binding.license.setOnClickListener(openExternalBrowser)
+        binding.repo.setOnClickListener(openExternalBrowser)
+        binding.privacy.setOnClickListener(openExternalBrowser)
+        binding.reportError.setOnClickListener(openExternalBrowser)
+        binding.rate.setOnClickListener(openExternalBrowser)
+        binding.credits.setOnClickListener { showCredits() }
+    }
+
+    private fun clearClickListeners() {
+        val binding = binding!!
+        binding.versionHistory.setOnClickListener(null)
+        binding.translate.setOnClickListener(null)
+        binding.license.setOnClickListener(null)
+        binding.repo.setOnClickListener(null)
+        binding.privacy.setOnClickListener(null)
+        binding.reportError.setOnClickListener(null)
+        binding.rate.setOnClickListener(null)
+        binding.credits.setOnClickListener(null)
+    }
+
+    private fun showCredits() {
+        val content = content!!
+        MaterialAlertDialogBuilder(this)
+            .setTitle(R.string.credits)
+            .setMessage(content.contributorInfo)
+            .setPositiveButton(R.string.ok, null)
+            .show()
+    }
+
+    companion object {
+        private const val TAG = "Catima"
+    }
+}
```

Comment:
Sorry for taking so long to get back to this. I've been thinking of this and I'm wondering, wouldn't it be cleaner if we remove the class level variables? That way we don't have to worry about manually nulling variables for garbage collection either and it severely simplifies the code:

```suggestion
package protect.card_locker

import android.os.Bundle
import android.view.MenuItem
import android.view.View
import com.google.android.material.dialog.MaterialAlertDialogBuilder
import protect.card_locker.databinding.AboutActivityBinding

class AboutActivity : CatimaAppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = AboutActivityBinding.inflate(layoutInflater)
        val content = AboutContent(this)

        title = content.pageTitle

        setContentView(binding.root)
        setSupportActionBar(binding.toolbar)
        enableToolbarBackButton()

        val copyright = binding.creditsSub
        copyright.text = content.copyright
        val versionHistory = binding.versionHistorySub
        versionHistory.text = content.versionHistory

        binding.versionHistory.tag = "https://catima.app/changelog/"
        binding.translate.tag = "https://hosted.weblate.org/engage/catima/"
        binding.license.tag = "https://github.com/CatimaLoyalty/Android/blob/master/LICENSE"
        binding.repo.tag = "https://github.com/CatimaLoyalty/Android/"
        binding.privacy.tag = "https://catima.app/privacy-policy/"
        binding.reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
        binding.rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"

        bindClickListeners(binding, content)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        val id = item.itemId
        if (id == android.R.id.home) {
            finish()
        }
        return super.onOptionsItemSelected(item)
    }

    private fun bindClickListeners(binding: AboutActivityBinding, content: AboutContent) {
        val openExternalBrowser = View.OnClickListener { view: View ->
            val tag = view.tag
            if (tag is String && tag.startsWith("https://")) {
                OpenWebLinkHandler().openBrowser(this, tag)
            }
        }

        binding.versionHistory.setOnClickListener(openExternalBrowser)
        binding.translate.setOnClickListener(openExternalBrowser)
        binding.license.setOnClickListener(openExternalBrowser)
        binding.repo.setOnClickListener(openExternalBrowser)
        binding.privacy.setOnClickListener(openExternalBrowser)
        binding.reportError.setOnClickListener(openExternalBrowser)
        binding.rate.setOnClickListener(openExternalBrowser)
        binding.credits.setOnClickListener { showCredits(content) }
    }

    private fun showCredits(content: AboutContent) {
        MaterialAlertDialogBuilder(this)
            .setTitle(R.string.credits)
            .setMessage(content.contributorInfo)
            .setPositiveButton(R.string.ok, null)
            .show()
    }
}
```

(I also seem to have accidentally made it so this PR needs rebasing, sorry)